### PR TITLE
Add zh-TW translation

### PR DESCRIPTION
**Pull request by roy4801**

### CODE REVIEW

File: i18n/zh-Hant.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "explore": "搜尋和探索",
+  "followers": "追隨者",
+  "following": "追隨",
+  "lists": "列表",
+  "profile": "個人資料",
+  "showNewMulti": "顯示 $ 新推文",
+  "showNewSingle": "顯示 1 個新推文",
+  "translateTweet": "翻譯推文",
+  "translatedTweetInfo": "由 $source$ 從 $lang$ 翻譯",
+  "tweets": "推文",
```
Comment (Bl4Cc4t) [inline i18n/zh-Hant.json:13]: Some of the default strings are not needed anymore, I found a way to include them natively
```suggestion
  "showNewMulti": "顯示 $ 新推文",
  "showNewSingle": "顯示 1 個新推文",
  "translatedTweetInfo": "由 $source$ 從 $lang$ 翻譯",
  
```

File: i18n/zh-Hant.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "keepTweetsInTL": "保留貼文 (目前沒有功能)",
+  "keepTweetsInTLDesc": "當貼文離開可視區時使網站不自動移除它",
+  "leftTrends": "側邊欄熱門話題",
+  "leftTrendsDesc": "在左側邊欄顯示熱門話題",
+  "show10Trends": "總是顯示 10 個熱門話題",
+  "stickySidebars": "固定側邊欄",
+  "smallSidebars": "小的側邊欄",
+  "smallSidebarsDesc": "使用 290px 的側邊欄 (預設 350px) ",
+  "squareAvatars": "正方形的頭像"
+}
```
Comment (Bl4Cc4t) [inline i18n/zh-Hant.json:32]:
```suggestion
  "squareAvatars": "正方形的頭像",
  "updateNotifications": "GoodTwitter 2 update notifications",

  "updatedInfo": "Updated to $version$!",
  "updatedInfoChangelog": "Changelog"
}

```

File: i18n/zh-Hant.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  "tweets": "推文",
+  "settingsHeaderTimeline": "時間軸",
+  "settingsHeaderSidebars": "側邊欄",
+  "settingsHeaderOther": "其他",
+  "biggerPreviews": "更大的圖片預覽",
+  "biggerPreviewsDesc": "長的圖片在推文內顯示完整的高度",
+  "disableAutoRefresh": "禁用自動刷新 (實驗性功能)",
+  "disableAutoRefreshDesc": "禁用時間軸自動貼文刷新 <br/>實驗性功能，結果可能會不正常",
+  "forceLatest": "強制最新貼文模式",
+  "forceLatestDesc": "最新貼文模式不再是永久啟用的。啟用這個選項後網站會自動切換回最新貼文模式",
```
Comment (Bl4Cc4t) [inline i18n/zh-Hant.json:22]:
```suggestion
  "forceLatestDesc": "最新貼文模式不再是永久啟用的。啟用這個選項後網站會自動切換回最新貼文模式",
  "hideTranslateTweetButton": "Hide Translate Tweet Button",
  "hideTrends": "Hide Trends",
  "hideWhoToFollow": "Hide Who To Follow",
```

**Comment by roy4801** [comment]
> Hm, I am not able to select that language, seems like they exclude a whole bunch of choices depending on your current location...
> So please excuse my dumb question, but is this [Taiwanese Mandarin](https://en.wikipedia.org/wiki/Zh-TW)?
> 
> Edit: I just saw the file name is zh-Hant, so this is Traditional Chinese? I am a bit confused...

Yes, it's Traditional Chinese and I'm from Taiwan so the language code is zh-TW.

**Comment by roy4801** [comment]
I have fixed the json file, please review it :)

**Comment by Bl4Cc4t** [comment]
> Yes, it's Traditional Chinese and I'm from Taiwan so the language code is zh-TW.

Hm okay then, but the `lang` attribute of the `html` element is `zh-Hant`?

**Comment by roy4801** [comment]
> Hm okay then, but the `lang` attribute of the `html` element is `zh-Hant`?

Yes, it's `zh_Hant` for Traditional Chinese (Taiwan).

**Comment by Bl4Cc4t** [comment]
That’s too bad, then I have to find some other method of detecting the language since `zh-Hant` is also set when using non-taiwanese Traditional Chinese

Can you please tell me what the loaded i18n scripts look like? (line 17 and 18)
image

**Comment by roy4801** [comment]
Here is the twitter explore page in Traditional Chinese (Taiwan)



https://pastebin.com/xaQ90Whf

**Comment by Bl4Cc4t** [comment]
Huh, so the i18n files are the same? So there is no difference between Traditional Chinese and Traditional Chinese (Taiwan)...

Sorry, I didn’t know that

**Pull request merged**